### PR TITLE
`run_layer_test` also runs symbolic calls when `input_data` is provided.

### PR DESCRIPTION
**Pull request by hertschuh**
This uncovered an issue in one layer and an issue in a test.

### CODE REVIEW

File: keras_core/testing/test_case.py [hunk 1]
```diff
@@ -405,6 +419,9 @@ def create_keras_tensors(input_shape, dtype, sparse):
             )
             for k, v in input_shape.items()
         }
+    raise ValueError(
+        f"Unsupported type for `input_shape`: {input_shape.__class__}"
```
Comment (fchollet) [inline keras_core/testing/test_case.py:423]: You can use `type(input_shape)` which will work with things that aren't instances

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

File: keras_core/testing/test_case.py (hunk 1 above)
Comment (hertschuh) [reply keras_core/testing/test_case.py]: Thanks! Fixed.

**Review by fchollet — APPROVED** [review]
LGTM

**Pull request merged**